x-pack/plugins/spaces/server/lib/create_spaces_service.ts

```diff
@@ -24,15 +26,25 @@ export function createSpacesService(server: any): SpacesService {
     return spaceId;
   }
 
+  function getIsDefaultSpaceId(request: any) {
+    return isDefaultSpaceId(getSpaceId(request));
```

Comment:
nit: I know how trivial this lookup is, but to be consistent with the rest of the SpacesService, we should be pulling this from the `contextCache`, rather than computing it every time.

We should also add tests to `create_spaces_service.test.ts` for the new public function